source/addComment.js

```diff
@@ -1,3 +1,27 @@
 $(document).ready(function(){
-  // add your code here
+
+  $('#new_comment_button').on('click',function(e){
+    e.preventDefault();
+    $list = $("#comment_list");
+
+    commentForm = $list.append('<form name="new_comment_form" id="new_comment_form">\
+        Name: <input type="text" name="name"><br>\
+        Comment: <textarea name="comment" form="new_comment_form"></textarea><br>\
+        <input type="submit" value="Submit">\
+        </form>');
+
+    $(commentForm).on('submit', function(e){
+      e.preventDefault();
+      attributes = ( $("#new_comment_form").serializeArray() );
```

Comment:
you shouldn't need to serializeArray here you can grab the input using something like this `'input[name=name]'`